source/api/content-state/0.9/index.md

```diff
@@ -230,13 +224,13 @@ For these reasons, this specification defines a two-step encoding that uses both
 
 **Note that "base64url" is not the same encoding as "base64".** 
 
-To encode:
+The process to encode a content state is:
 
 * encode to a UTF-8 string described by [encodeURI][org-ecma-encodeuri]
 * encode the resulting UTF-8 string as [base64url][org-rfc-4648-5]
 * remove any "=" padding characters from the end
 
-To decode:
+Conversely to decode a content state:
 
 * restore any missing "=" padding characters to the end of the string
```

Comment:
Should this be specific about how to tell what "=" characters need be added? The "missing" ones seems vague. Perhaps:
* restore any removed "=" padding characters to the end of the string to make it a multiple of 4 characters long